dags/scripts/full-scrape.sh

```diff
@@ -1,11 +1,25 @@
-#!/bin/sh
+#!/bin/bash
 set -e
 
+import_to_staging() {
+    # If the database URL does not contain "staging", it's the production
+    # database. Append "_staging" and import. Assumes that the staging and
+    # production databases live on the same server, and follow the naming
+    # convention "${DATABASE_NAME}" and "${DATABASE_NAME}_staging".
+    if [[ "$DATABASE_URL" != *"staging"* ]]; then
+        STAGING_DATABASE_URL="${LA_METRO_DATABASE_URL}_staging"
+        echo "Importing into staging database ${STAGING_DATABASE_URL}"
+        SHARED_DB=True DATABASE_URL=$STAGING_DATABASE_URL pupa update lametro --import || echo "${STAGING_DATABASE_URL} does not exist"
+    fi
+}
```

Comment:
This is a clever function, but it seems risky to hardcode the assumptions that A) the databases are named a certain way and B) they live on the same server. What if we introduced a new env var, `LA_METRO_STAGING_DATABASE_URL`, and only run `pupa update` if the var is not empty?

Reply:
Great call, @jeancochrane. Made this change.